hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -297,6 +297,10 @@ def get_different_seeds_hyperdrive_config(self) -> HyperDriveConfig:
 
 
 class DatasetParams(param.Parameterized):
+    INPUT_0: List[str] = param.List(default=[], class_=str, doc="Do not use. For AML SDK v2 only")
```

Comment:
Didn't we say that we don't need these anymore?

Reply:
Good point - this has been removed and replaced with argument filtering at the start of the script